### PR TITLE
Create dashboard onboard

### LINKED ISSUE #3: Dashboard - No Ratings
As an authenticated user, who has rated fewer than 5 cocktails, I see a dashboard with a selection of cocktails to rate

When I rate a cocktail, the number at the top of the screen goes down.

estimated completion 7/30

### PR DESCRIPTION
**Pull request by zachjamesgreen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Linter has passed locally and any fixes were made for failures
- [x] Build is passing
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
## What is the current behavior?

Issue Number: 3
closes #3 
## What is the new behavior?

- 
- 
- 
## Does this introduce a breaking change?
- [ ] Yes
- [x] No
there is an issue with hovering of the stars on the onboard page. made an issue to deal with later.

### CODE REVIEW

File: src/components/search_results.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+const queryString = require('query-string');
+const _ = require('lodash');
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {query: this.parseParams(props)}
```
Comment (markcyen) [inline src/components/search_results.js:8]: Interesting code... Can't wait to understand what this does.

File: src/components/star.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    this.state = {filled: false}
+  }
+
+  render() {
+    return(
+        <svg 
+          data-star-id={this.props.starId}
+          className={`${this.props.marked ? 'text-black' : ''} hover:fill-current transition`}
+          width="26" height="23" viewBox="0 0 26 23" fill={this.props.marked ? 'black' : 'none'} xmlns="http://www.w3.org/2000/svg">
+          <path d="M12.8567 3.15498L14.7947 8.95425L15.023 9.63729H15.7431H21.9352L16.9501 13.1587L16.3431 13.5875L16.5787 14.2924L18.4978 20.0351L13.4336 16.4578L12.8567 16.0502L12.2797 16.4578L7.21548 20.0351L9.13464 14.2924L9.37021 13.5875L8.76316 13.1587L3.77813 9.63729H9.97015H10.6903L10.9186 8.95425L12.8567 3.15498Z" stroke="black" strokeWidth="2"/>
```
Comment (markcyen) [inline src/components/star.js:15]: This looks like the code to hover over the stars. Interesting stuff!

**Review by markcyen — APPROVED** [review]
Super cool React and JS code!

**Review by twarbelow — APPROVED** [review]
this looks good
in a future iteration, let's have a visual indication that the user has focused on the search input field

**Pull request merged**